.github/CONTRIBUTING.md

```diff
@@ -274,7 +274,13 @@ Math operators like +, -, /, *, etc are up in the air, just choose which version
 #### Use
 * Bitwise AND - '&'
 	* Should be written as ```bitfield & bitflag``` NEVER ```bitflag & bitfield```, both are valid, but the latter is confusing and nonstandard.
-
+* Do not explicitly assign `null` to newly declared variables as this is already their default value
```

Comment:
Strongly disagree with this. 

Yeah, it is their default state, but it's clearer when it's explicitly null. It doesn't make things look ugly and it helps cement it in people's mind that it's a holder.

It's far easier to glance at an datum/turf/mob/object and see what the holders are by `= null`. It's especially useful for tracking down other references, very quickly.